robotiq_force_torque_sensor/nodes/rq_sensor.cpp

```diff
@@ -50,13 +50,55 @@
 #include "robotiq_force_torque_sensor/ft_sensor.h"
 #include "robotiq_force_torque_sensor/sensor_accessor.h"
 
+typedef robotiq_force_torque_sensor::sensor_accessor::Request Request;
+
 static void decode_message_and_do(INT_8 const  * const buff, INT_8 * const ret);
+static void decode_message_and_do(robotiq_force_torque_sensor::sensor_accessor::Request& req,
+								  robotiq_force_torque_sensor::sensor_accessor::Response& res);
+
 static void wait_for_other_connection(void);
 static int max_retries_(100);
 
 ros::Publisher sensor_pub_acc;
 
 /**
+ * @brief decode_message_and_do Decode the message received and do the associated action
+ * @param req
+ * @param res
+ */
+static void decode_message_and_do(robotiq_force_torque_sensor::sensor_accessor::Request& req,
```

Comment:
I didn't realize you could have a service with this signature.  I thought a service always had to return a bool (like [here](http://wiki.ros.org/ROS/Tutorials/WritingServiceClient%28c%2B%2B%29#roscpp_tutorials.2BAC8-Tutorials.2BAC8-WritingServiceClient.CA-51b90df3945605c351da24a5ec712323d4d3dc5a_4).